test/rpc-test.js

```diff
@@ -0,0 +1,227 @@
+    await assert.asyncThrows( async () =>
+      await wclient.execute('sendmany', ['default', sendTo]),
+      'Invalid amount.'
+    );
+  });
+
+  it('should rpc listreceivedbyaddress', async () => {
+    await wclient.execute('selectwallet', ['hot']);
+
+    const listZeroConf = await wclient.execute('listreceivedbyaddress',
+      [0, false, false]);
+    assert.deepStrictEqual(listZeroConf, [{
+      'involvesWatchonly': false,
+      'address': addressHot,
+      'account': 'default',
+      'amount': 1.1234,
+      'confirmations': 0,
+      'label': ''
+    }]);
+
+    blocks.push(await nclient.execute('generatetoaddress', [1, addressMiner]));
+    await wdb.syncChain();
+
+    const listSomeConf = await wclient.execute('listreceivedbyaddress',
+      [1, false, false]);
+    assert.deepStrictEqual(listSomeConf, [{
+      'involvesWatchonly': false,
+      'address': addressHot,
+      'account': 'default',
+      'amount': 1.1234,
+      'confirmations': 1,
+      'label': ''
+    }]);
+
+    const listTooManyConf = await wclient.execute('listreceivedbyaddress',
+      [100, false, false]);
+    assert.deepStrictEqual(listTooManyConf, []);
+  });
+
+  it('should rpc listtransactions', async () => {
+    const txs = await wclient.execute('listtransactions', []);
+    assert.strictEqual(txs.length, 2);
+    assert.strictEqual(txs[0].amount + txs[1].amount, 1.1234);
+    assert.strictEqual(txs[0].account, 'default');
+  });
+
+  it('should rpc listunspent', async () => {
+    utxo = await wclient.execute('listunspent', []);
+    assert.strictEqual(utxo.length, 2);
+  });
+
+  it('should rpc lockunspent and listlockunspent', async () => {
+    let result = await wclient.execute('listlockunspent', []);
+    assert.deepStrictEqual(result, []);
+
+    // lock one utxo
+    const output = utxo[0];
+    const outputsToLock = [{'txid': output.txid, 'vout': output.vout}];
+    result = await wclient.execute('lockunspent', [false, outputsToLock]);
+    assert(result);
+
+    result = await wclient.execute('listlockunspent', []);
+    assert.deepStrictEqual(result, outputsToLock);
+
+    // unlock all
+    result = await wclient.execute('lockunspent', [true]);
+    assert(result);
+
+    result = await wclient.execute('listlockunspent', []);
+    assert.deepStrictEqual(result, []);
+  });
+
+  it('should rpc listsinceblock', async () => {
+    const listNoBlock = await wclient.execute('listsinceblock', []);
+    assert.strictEqual(listNoBlock.transactions.length, 2);
+    // txs returned in unpredictable order
+    const txids = [
+      listNoBlock.transactions[0].txid,
+      listNoBlock.transactions[1].txid
+    ];
+    assert(txids.includes(txid));
+
+    const block5 = blocks[5];
+    const listOldBlock = await wclient.execute('listsinceblock', [block5]);
+    assert.strictEqual(listOldBlock.transactions.length, 2);
+
+    const nonexistentBlock = consensus.ZERO_HASH.toString('hex');
+    await assert.asyncThrows( async () =>
```

Comment:
nit: This may read better as:
```js
await assert.asyncThrows(async () => {
  await wclient.execute('listsinceblock', [nonexistentBlock])
}, 'Block not found');
```

Similar with the others above.